onnxruntime/core/graph/function_utils.cc

```diff
@@ -415,7 +290,7 @@ std::unique_ptr<ONNX_NAMESPACE::OpSchema> CreateSchema(const std::string& functi
   const auto onnx_released_versions =
       schema_registry.GetLastReleasedOpsetVersions(false);
 
-  InlinedHashMap<std::string, int> func_domain_to_version;
+  std::unordered_map<std::string, int> func_domain_to_version;
```

Comment:
Not relevant to this PR: I am curious that whether in the future ONNX can provide flexible interface to let ORT use InlinedHashMap (actually absl::flat_hash_map), if the memory save is worthy.